pkg/apis/storage/validation/validation_test.go

```diff
@@ -1726,6 +1726,49 @@ func TestCSIDriverValidation(t *testing.T) {
 				PodInfoOnMount: &notPodInfoOnMount,
```

Comment:
I realized we're missing a unit test for `ValidateCSIDriverUpdate`. Could you look into adding one that tries to modify volumeModes?

Reply:
Okay.

Reply:
Still open, will follow shortly.

Reply:
Done, in a separate commit because it isn't directly related to some of the other changes.